package.json

```diff
@@ -43,14 +43,15 @@
     "csv-parse": "1.1.10",
     "expect": "1.20.2",
     "ezs": "1.1.3",
-    "ezs-basics": "2.2.1",
+    "ezs-basics": "2.2.5",
     "file-loader": "0.9.0",
     "html-webpack-plugin": "2.26.0",
     "inist-ark": "2.0.0",
     "isomorphic-fetch": "2.2.1",
     "jbj": "7.1.4",
     "json-csv": "1.3.0",
     "json-loader": "0.5.4",
+    "jsonld": "^0.4.11",
```

Comment:
ok, so we can use a per-project config file - /path/to/my/project/.npmrc 
is it a good idea ?